Source/Scene/DeviceOrientationCameraController.js

```diff
@@ -0,0 +1,106 @@
+/*global define*/
+define([
+           '../Core/defaultValue',
+           '../Core/defined',
+           '../Core/destroyObject',
+           '../Core/DeveloperError',
+           '../Core/Math',
+           '../Core/Matrix3',
+           '../Core/Quaternion'
+       ], function(
+           defaultValue,
+           defined,
+           destroyObject,
+           DeveloperError,
+           CesiumMath,
+           Matrix3,
+           Quaternion) {
+    "use strict";
+
+    function DeviceOrientationCameraController(scene) {
```

Comment:
I plan to keep it `@private`. This should be expanded to use a [PositionSensorVRDevice](https://developer.mozilla.org/en-US/docs/Web/API/PositionSensorVRDevice) when we fully support WebVR. Still, there isn't anything configurable there so I don't know if we would every expose it.